experiment/example/example.go

```diff
@@ -0,0 +1,63 @@
+// Package example contains a simple example of experiment.
+package example
+
+import (
+	"context"
+	"time"
+
+	"github.com/ooni/probe-engine/experiment"
+	"github.com/ooni/probe-engine/experiment/handler"
+	"github.com/ooni/probe-engine/model"
+	"github.com/ooni/probe-engine/session"
+)
+
+const (
+	testName    = "example"
+	testVersion = "0.0.1"
+)
+
+// Config contains the experiment config.
+//
+//This includes all variables that will be used inside or outside the module.
+type Config struct {
+	SleepTime time.Duration
+}
+
+// TestKeys contains the experiment's result.
+//
+// This is what will end up into the Measurement.TestKeys field
+// when you run this experiment.
+// In other words, the variables in this struct will be
+// the particular results of this experiment.
+type TestKeys struct {
+	Success bool `json:"success"`
+}
+
+// This is the main function of each experiment,
+// that is called by the NewExperiment.
+func measure(
+	ctx context.Context, sess *session.Session, measurement *model.Measurement,
+	callbacks handler.Callbacks, config Config,
+) error {
+	testkeys := &TestKeys{Success: true}
+	measurement.TestKeys = testkeys
+	time.Sleep(config.SleepTime)
+	return nil
+}
+
+// NewExperiment creates a new experiment.
+// This is the preliminary function that is called from outside the module.
```

Comment:
Maybe "This is the function that you call to create an instance of this experiment. Once you have created an instance, you can use directly the generic experiment API."

Reply:
Although this comment is still a little difficult to understand, it's better than my comment. :P

Reply:
🤣